src/common/utils.tsx

```diff
@@ -26,6 +28,23 @@ interface QrCode {
   };
 }
 
+export function readFileAsCsv<T>(file: File, headers?: string[]): Promise<Array<T>> {
```

Comment:
generic type T here is a bit weird, a brief look into the documentation doesn't really tell me what the return type of csv().fromString is

Reply:
Done.